src/main/resources/view/MonthDeliveryJobListPanel.fxml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.TextArea?>
+<?import javafx.scene.control.TreeView?>
+<?import javafx.scene.layout.VBox?>
+
+
+<VBox maxHeight="1.7976931348623157E308" prefHeight="500.0" prefWidth="248.0" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <TreeView fx:id="monthDeliveryJobListView" prefHeight="424.0" prefWidth="238.0">
+         <VBox.margin>
+            <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+         </VBox.margin>
+         <padding>
+            <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+         </padding>
+      </TreeView>
+      <TextArea prefHeight="200.0" prefWidth="200.0" text="helllllllllllllllllllllloooooo" />
+   </children>
```

Comment:
👋

Reply:
oh right LOL let me delete that part i was testing